code/modules/mob/inventory.dm

```diff
@@ -43,6 +43,8 @@
 	Returns TRUE on success.
 */
 /mob/proc/put_in_l_hand(obj/item/W)
+	if(isturf(W.loc))
+		W.do_pickup_animation(src)
```

Comment:
why add these isturf checks here and put in r hand if its already checked in the proc?